MediaPicker/src/main/java/org/wordpress/android/mediapicker/MediaPickerActivity.kt

```diff
@@ -216,6 +216,7 @@ class MediaPickerActivity : LocaleAwareActivity(), MediaPickerListener {
             IMAGE_EDITOR_EDIT_IMAGE -> {
                 data?.let {
                     val intent = Intent()
+                    // TODO: 20/07/2021 There's a whole ImageEditor module in WPAndroid. Should we import it?
```

Comment:
@anitaa1990 do you think we should also add `ImageEditor` to the `MediaPicker`? If not (or not in the first iteration) we will need to provide some interfaces to glue together `MediaPicker` and `ImageEditor` from WPAndroid.